rotkehlchen/chain/gnosis/transactions.py

```diff
@@ -11,13 +14,45 @@
 from .modules.xdai_bridge.constants import BLOCKREWARDS_ADDRESS
 
 if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.types import LOG_ITERATION_CALLBACK_ARGUMENTS_TYPE
     from rotkehlchen.db.dbhandler import DBHandler
 
     from .node_inquirer import GnosisInquirer
 
 ADDED_RECEIVER_ABI: Final = [{'anonymous': False, 'inputs': [{'indexed': False, 'name': 'amount', 'type': 'uint256'}, {'indexed': True, 'name': 'receiver', 'type': 'address'}, {'indexed': True, 'name': 'bridge', 'type': 'address'}], 'name': 'AddedReceiver', 'type': 'event'}]  # noqa: E501
 DEPLOYED_BLOCK: Final = 9053325
 DEPLOYED_TS: Final = 1539027985
+NO_BLOCK_PROCESSED_VALUE = 0
```

Comment:
0 is a valid block value

```suggestion
NO_BLOCK_PROCESSED_VALUE: Final = -1
```